### DISCUSSION TITLE
Live kali Linux USB for MacBook M1

### DISCUSSION THREAD

**Discussion by Draih79** [tracelabs/tlosint-live#156] [Q&A]
Hi guys. I am sorry if this question has been answered before but I am trying to make a kali Linux live usb which works with MacBook M1. Is there any solution to this? Thank you

**Comment by humandecoded** [comment]
We do a build for M1 macs: https://download.tracelabs.org/TL-OSINT-2022.1-MAC-M1.iso

Should be able to write that to USB via Etcher (in on mac).
